scrapers/chastitybabes.yml

```diff
@@ -0,0 +1,47 @@
+name: "ChastityBabes"
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - 'chastitybabes.com/archives'
+    scraper: sceneScraper
+    
+xPathScrapers:
+  sceneScraper:
+    common:
+      $performerTitle: //h1[@id="post-title"]/text()
+    scene:
+      Title:
+        selector: $performerTitle
+        postProcess:
+          - replace:
+            - regex: (.*) – (.*)
+              with: $2
+      Date: 
+        selector: //div[@class="post_info"]/text()[1]
+        postProcess:
+          - replace:
+            - regex: '.*Posted on (.*) in.*'
+              with: $1
+          - parseDate: "Jan 2, 2006" 
+      Details: 
+        selector: //div[@class="postcontent"]/p/text()
+        concat: "\n"
+      Performers: 
+        Name: 
+          selector: $performerTitle
+          postProcess:
+            - replace:
+              - regex: (.*) – (.*)|.*
+                with: $1
+      Tags: 
+        Name: //a[@rel="category tag"]/text()
+      Studio:
+        Name:
+          selector: //title/text()
+          postProcess:
+            - replace: 
+              - regex: .* \| (.*)
+                with: $1
```

Comment:
Is there a case where the studio isnt `Chastity Babes` ?
If not then a fixed should be used instead eg
```yaml
          fixed: Chastity Babes
```

Reply:
I know the parent studio has more sites and the updates on chastitybabes aren't sequentially numbered, so I'm guessing there may be more sites using the same layout. I dont know which though.

Reply:
Oh you mean more sites could be added to the scraper later on. Yeah that makes sense.

Reply:
I checked 14 sites by the same parent studio (all I could find) and they all have different layouts that share barely anything. So I made the studio name fixed as suggested